content/hardware/02.hero/boards/uno-rev3/tutorials/PWM-using-MATLAB-Tutorial/PWM-using-MATLAB-Tutorial.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Controlling PWM output with a potentiometer on the Arduino Uno via MATLAB'
+difficulty: intermediate
+compatible-products: [Arduino Uno Rev 3, Potentiometer]
+description: 'Learn how to dim an LED connceted to the Arduino Uno with MATLAB.'
+tags:
+  - MATLAB
+author: 'Mohammed Hussain Jahangiri'
+hardware:
+  - hardware/boards/arduino-uno-rev3
+software:
+  - MATLAB
+---
+
+## Introduction
+
+In this tutorial, we will use MATLAB to turn on an external LED, and then to adjust the output using MATLAB® Support Package for Arduino® Hardware.
+
+
+### Goals
+
+The goals of this project are:
+
+- Control the Arduino Uno LED with MATLAB.
+- Control the Arduino Uno PWM output with MATLAB.
+- Create a dimmable light using a potentiometer.
+
+### Hardware & Software Needed
+
+- [Arduino Uno Rev3](https://store.arduino.cc/products/arduino-uno-rev3)
+- [USB 2.0 Cable Type A/B](https://store.arduino.cc/products/usb-2-0-cable-type-a-b)
+- Any potentiometer connectable to a breadboard
+- Valid MATLAB® licence
+- [MATLAB® Support Package for Arduino® Hardware](https://www.mathworks.com/matlabcentral/fileexchange/47522-matlab-support-package-for-arduino-hardware)
+
+
+***The Arduino Uno, potentiometer, USB 2.0 Cable Type A/B and jumper wires are included as part of the [Arduino Starter Kit](https://store.arduino.cc/collections/kits/products/arduino-starter-kit-multi-language)***
+
+***A valid MATLAB® licence is needed. Your workplace or education institution may have a subscription. Alternatively, a one-year trial subscription to MATLAB® is included as part of the [Arduino Engineering Kit R2](https://store.arduino.cc/products/arduino-engineering-kit-rev2).***
+
+
+### Why MATLAB?
+MATLAB® is an educational and industrial programming platform used to analyse data, perform simulations and carry out model based designs. Through an interactive communication with an Arduino board, you can expand MATLAB's capabilities, while also gaining access to a wide range of math, engineering and plotting functions. Check out the capabilities of MATLAB® over on the [MathWorks website](https://www.mathworks.com/solutions.html#capabilities). In this tutorial, we will establish the connection with MATLAB via a USB and Wi-Fi connection. The [MATLAB Support Package for Arduino Hardware](https://www.mathworks.com/matlabcentral/fileexchange/47522-matlab-support-package-for-arduino-hardware) must be installed (which may require a MathWorks account), in order to facilitate the communication between the MATLAB software on your computer and your Arduino board. 
+ 
+
+### LED on the Arduino Uno R3
+The internal LED in the Arduino Uno is connected to Digital Pin 13. By pulling this pin HIGH, we can turn the LED on and by pulling it LOW we can turn the LED off. But to output a variable voltage we have to use pins such as pin ~11 or ~10.
+
+<p align="center">
+<img src="http://electronicswithyou.com/wp-content/uploads/2019/05/Arduino-Uno-LED.jpg"/>
```

Comment:
Please use the markdown formatting instead, for example:

![image description](link.com/url/to/image.jpg)

Reply:
Will do.